src/test/kotlin/no/nav/dagpenger/inntekt/db/PostgresTest.kt

```diff
@@ -0,0 +1,100 @@
+package no.nav.dagpenger.inntekt.db
+
+import com.zaxxer.hikari.HikariDataSource
+import no.nav.dagpenger.inntekt.Configuration
+
+import no.nav.dagpenger.inntekt.inntektskomponenten.v1.Aktoer
+import no.nav.dagpenger.inntekt.inntektskomponenten.v1.AktoerType
+import no.nav.dagpenger.inntekt.inntektskomponenten.v1.InntektkomponentenResponse
+import no.nav.dagpenger.inntekt.v1.InntektRequest
+import org.junit.Test
+import org.testcontainers.containers.PostgreSQLContainer
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import kotlin.test.assertTrue
+
+internal class PostgresTest {
+
+    @Test
+    fun `Migration scripts are applied successfully`() {
+        withCleanDb {
+            val migrations = migrate(DataSource.instance)
+            assertEquals(1, migrations, "Wrong number of migrations")
+        }
+    }
+
+    @Test
+    fun `Migration scripts are idempotent`() {
+        withCleanDb {
+            migrate(DataSource.instance)
+
+            val migrations = migrate(DataSource.instance)
+            assertEquals(0, migrations, "Wrong number of migrations")
+        }
+    }
+
+    @Test
+    fun `JDBC url is set correctly from  config values `() {
+        with(hikariConfigFrom(Configuration())) {
+            assertEquals("jdbc:postgresql://localhost:5432/dp-inntekt-db", jdbcUrl)
+        }
+    }
+}
+
+internal class PostgresInntektStoreTest {
+    @Test
+    fun `Successful insert of inntekter`() {
```

Comment:
Perhaps test that the mapping table has a a row as well?

Reply:
good point!